.github/workflows/tests.yml

```diff
@@ -179,7 +179,7 @@ jobs:
         run: inv -r faasmcli/faasmcli python.clear-runtime-pyc
       # --- Test run ---
       - name: "Run the tests"
-        run: /build/faasm/bin/tests
+        run: /build/faasm/bin/tests --abort
```

Comment:
This makes the tests exit at the first failure. Otherwise it is sometimes hard to see which test has failed.